doc/howto.md

```diff
@@ -0,0 +1,102 @@
+# How-Tos
+
+## Concepts
+
+- `baseline`, one of the best KPI records in the history, the baseline should keep being updated.
```

Comment:
should this be a running average or something similar? I'm afraid that the "best" one will be too good for the "normal" ones. But we can revisit this in the future.